### ISSUE TITLE
Frequent request to solve captchas after latest update

### ISSUE THREAD

**Issue by happy-420** [PokeAlert/PokeAlert#323]
I'm not sure if it's just me, but after installing the latest update (PokeAlert 4.0.5), accounts seem to request to solve captchas more frequently than before.

Any tips for this? Thanks a lot!

**Comment by rezecib** [comment]
I've been getting way more as well.

**Comment by battosai50** [comment]
Yeah it really is a problem. 60 accounts...59 need captcha again and again. Can't use pokealert anymore. Fyi: 50% of them were created with ancient api. I checked the new "Real" behaviour option...maybe it's what causes the problem. Please solve this as it ruins the app actually

**Comment by battosai50** [comment]
Redownloaded lastest pokealert apk, réinstalled, unchecked Real behaviour option, recaptcha all and for now i havent been asked to captcha again. Fingers crosses...

**Comment by Eric678** [comment]
For me turning on Ditto botting has pretty much fixed the problem - all my accounts are above L7 now and captchas have dropped significantly!
